### github에서 새로운 repository를 생성하고, readme 파일에 간단한 설명을 적었다. 이후 로컬에서 git init, add, commit 을 실행하고, git push 명령어를 이용하여 로컬 레포지토리의 내용을 github 원격 레포지토리로 push하려 했으나 다음과 같은 오류가 발생하였다.
```
error : failed to push some refs to '~~~'
hint : Updates were rejected bacause the remote contains work that you do not have locally.
```
##### 원인을 파악하고 해결(조치) 방법을 제시하시오.

- 발생원인 : git hub와 로컬 저장소가 동기화 되어 있지 않기 때문이다. 동기화 시켜주면 간단하게 해결 할 수 있다.
- 해결방안 : **git pull origin master or git pull**
